libdevcore/vector_ref.h

```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+#include "db.h"
```

Comment:
This vector_ref.h file uses `DEV_LDB` macro defined in db.h
This came up after reordering of #include's in another file